microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -44,12 +45,13 @@ These experiments could be unstable or unreliable and may require you to restart
 ## Experiments which are turned on by default
 
 The following experimental features are turned on by default. You can use these features right away, without changing any settings. You can turn off these default experimental features, if needed.
+<!-- todo: the defaults vary per user.  how much?  which? eg focus mode -->
```

Comment:
@captainbrosset - Can we write a general note here that defaults may vary per user?

Reply:
I had no idea that the defaults varied per users. Can you tell me more about this?

Reply:
This would largely be the case for users trying experimental features. I think it's OK to leave this nuance out of the docs, as any user trying, for example, the Focus Mode experience will get an opt-in/opt-out prompt that updates their settings selections.

Reply:
removed comment